Cargo.toml

```diff
@@ -51,7 +51,7 @@ blake2b_simd = "1.0"
 cid = { version = "0.8", default-features = false, features = ["std"] }
 frc42_dispatch = "3.0.0"
 frc46_token = "3.1.0"
-fvm = "~2.2"
+fvm = "2.2"
```

Comment:
The FVM does not follow semantic versioning. Version `2.2` is not guaranteed to be in any way compatible with version `2.3`. Let's bump the dependency to `~2.3`.

Reply:
fixed